sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveUserDefinedTypeSuite.scala

```diff
@@ -18,26 +18,28 @@
 package org.apache.spark.sql.hive
 
 import scala.collection.JavaConverters._
+import scala.util.Random
 
 import org.apache.hadoop.hive.ql.udf.generic.GenericUDF
 import org.apache.hadoop.hive.serde2.objectinspector.{ObjectInspector, StandardListObjectInspector}
 import org.apache.hadoop.hive.serde2.objectinspector.primitive.PrimitiveObjectInspectorFactory
 
-import org.apache.spark.sql.{QueryTest, RandomDataGenerator, Row}
+import org.apache.spark.sql.{QueryTest, Row}
 import org.apache.spark.sql.catalyst.FunctionIdentifier
 import org.apache.spark.sql.hive.test.TestHiveSingleton
 import org.apache.spark.sql.test.{ExamplePoint, ExamplePointUDT}
 import org.apache.spark.sql.types.StructType
 
+
 class HiveUserDefinedTypeSuite extends QueryTest with TestHiveSingleton {
   private val functionClass = classOf[org.apache.spark.sql.hive.TestUDF].getCanonicalName
 
   test("Support UDT in Hive UDF") {
+    val rand = new Random
     val functionName = "get_point_x"
     try {
-      val schema = new StructType().add("point", new ExamplePointUDT)
-      val inputGenerator = RandomDataGenerator.forType(schema, nullable = false).get
-      val input = inputGenerator.apply().asInstanceOf[Row]
+      val schema = new StructType().add("point", new ExamplePointUDT, nullable = false)
+      val input = Row.fromSeq(Seq(new ExamplePoint(rand.nextDouble(), rand.nextDouble())))
```

Comment:
I think you can just pick two numbers for `ExamplePoint` but don't bother with randomeness.

Reply:
Yea, better to remove the randomness...

Reply:
Yeah I've just tried to follow origin intention as it was trying to provide randomness. Looks like the test doesn't really deal with randomness. Will just pick any numbers.